packages/next/src/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const KIB = 1024;
```

Comment:
Moved these to its own file because there were some cyclic imports in `utils.ts` which esbuild was having some issues with.